pastepwn/analyzers/databasedumpanalyzer.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from .basicanalyzer import BasicAnalyzer
+
+
+class DatabaseDumpAnalyzer(BasicAnalyzer):
```

Comment:
Since you are using a regex to match the paste, you can easily subclass the regex analyzer. Otherwise the match function will rise an exception.

Reply:
Fixed